docs/docs/performance.md

```diff
@@ -9,32 +9,39 @@ This means that performance inside an enclave will normally be lower than with a
-| empty         |   Host     | 49453365.793 | ±  3404118.758 |  ops/s |
-| binary_trees  |   Enclave  |      454.061 | ±       31.089 |  ops/s |
-| binary_trees  |   Host     |     1758.980 | ±       79.428 |  ops/s |
-| fannkuch      |   Enclave  |        4.181 | ±        0.024 |  ops/s |
-| fannkuch      |   Host     |        5.925 | ±        0.063 |  ops/s |
-| fasta         |   Enclave  |        3.185 | ±        0.028 |  ops/s |
-| fasta         |   Host     |        4.022 | ±        0.127 |  ops/s |
-| himeno        |   Enclave  |        0.179 | ±        0.004 |  ops/s |
-| himeno        |   Host     |        0.366 | ±        0.003 |  ops/s |
-| mandelbrot    |   Enclave  |        5.529 | ±        0.134 |  ops/s |
-| mandelbrot    |   Host     |        6.385 | ±        0.132 |  ops/s |
-| nbody         |   Enclave  |        1.205 | ±        0.021 |  ops/s |
-| nbody         |   Host     |        1.279 | ±        0.017 |  ops/s |
-| pidigits      |   Enclave  |        9.941 | ±        0.185 |  ops/s |
-| pidigits      |   Host     |       24.722 | ±        0.301 |  ops/s |
-| spectral_norm |   Enclave  |       11.923 | ±        0.274 |  ops/s |
-| spectral_norm |   Host     |       17.345 | ±        0.930 |  ops/s |
+| Benchmark    | SGX   | Score (ops/s) | Score Delta (%) | Error (ops/s)  | Error (%) |
+|--------------|-------|---------------|-----------------|----------------|-----------|
+| empty        | 	Off	| 24892554.58	| N.A.            | ±7868.489      | ±0.03%    |
+| empty        | 	On	| 23388.662	    | -99.91%         | ±846.568	   | ±3.62%    |
+| binary_trees | 	Off	| 1508.331		| N.A.            | ±1.312	       | ±0.09%    |  
+| binary_trees | 	On	| 669.865	    | -55.59%         | ±8.807	       | ±1.31%    |
+| fannkuch     | 	Off	| 5.092		    | N.A.            | ±0.008	       | ±0.16%    |
+| fannkuch     | 	On	| 4.588	        | -9.90%          | ±0.005	       | ±0.11%    |
+| fasta        | 	Off	| 3.516	        | N.A.            | ±0.003	       | ±0.09%    |
+| fasta        | 	On	| 2.847	        | -19.03%         | ±0.002	       | ±0.07%    |
+| himeno       | 	Off	| 0.398	        | N.A.            | ±0.001	       | ±0.25%    |
+| himeno       | 	On	| 0.264	        | -33.67%         | ±0.003	       | ±1.14%    |
+| mandelbrot   | 	Off	| 5.039	        | N.A.            | ±0.001	       | ±0.02%    |
+| mandelbrot   | 	On	| 4.525	        | -10.20%	      | ±0.001	       | ±0.02%    |
+| nbody        | 	Off	| 1.384	        | N.A.            | ±0.002	       | ±0.14%    |
+| nbody        | 	On	| 1.412	        | 2.02%	          | ±0.001	       | ±0.07%    |
+| pidigits     | 	Off	| 19.434	    | N.A.            | ±0.011	       | ±0.06%    |
+| pidigits     | 	On	| 14.981	    | -22.91%	      | ±0.026	       | ±0.17%    |
+| spectral_norm| 	Off	| 12.843		| N.A.            | ±0.004	       | ±0.03%    |
+| spectral_norm| 	On	| 9.688	        | -24.57%	      | ±0.004	       | ±0.04%    |
```

Comment:
Ignore the formatting shown by git. If you open on IntelliJ, it will look correct.